packages/data-grid/src/quickFilter/QuickFilter.jsx

```diff
@@ -27,20 +28,40 @@ const propTypes = {
   ).isRequired,
   /** Function called whenever a user clicks an action */
   onFilterSelect: func.isRequired,
+  /** Text which will be displayed if there are no options to filter by */
+  emptyFilterLabel: string,
 };
 
-function QuickFilter({ filters, onFilterSelect }) {
+const defaultProps = {
+  emptyFilterLabel: 'No items to filter by',
+};
+
+function QuickFilter({ filters, onFilterSelect, emptyFilterLabel }) {
+  /** Used for the options array when there is no items to filter by */
+  const emptyFilterOption = [
+    {
+      label: emptyFilterLabel,
+      disabled: true,
+      value: '',
+    },
+  ];
+
   return (
     <div className="dg-quick-filter-container">
       <div className="dg-quick-filter-filters">
         {filters.map((filter, idx) => {
           return (
             <ButtonSelect
-              className="dg-quick-filter"
+              className={classnames('dg-quick-filter-filter', {
+                'dg-quick-filter': true,
```

Comment:
If this is always going to be true you can just have it declared in the function instead of within an object and with logic. classnames('dg-quick-filter-filter', 'dg-quick-filter', {'dg-quick-filter-empty': filter.options.length === 0, })}

Reply:
Thanks @Lukeaber - This has now been updated along with the added README file which is hopefully up to standard 😁